src/huggingface_hub/file_download.py

```diff
@@ -279,11 +351,10 @@ def cached_download(
         except (
             requests.exceptions.ConnectionError,
             requests.exceptions.Timeout,
+            OfflineModeIsEnabled,
         ) as exc:
             # Actually raise for those subclasses of ConnectionError:
-            if isinstance(exc, requests.exceptions.SSLError) or isinstance(
-                exc, requests.exceptions.ProxyError
-            ):
+            if isinstance(exc, requests.exceptions.ProxyError):
```

Comment:
was this intended to remove this?

Reply:
Yes otherwise the test with the bogus url test fails (raises SSLError while ValueError is expected)